### ISSUE TITLE
Step Assist Trigger

### ISSUE THREAD

**Issue by Fortist** [pWn3d1337/Techguns#40]
Like the majority of step assist mods/enchants/features they work automatically, however, yours works differently and I cant seem to find the trigger to step up a block consistently.
Looking up works some of the time, 
But it doesnt work in small spaces. This is probably a feature. A dumb one, however.
It also seems like having a 'running start' helps, which again doesnt really work for small spaces.
While Im glad there is a trigger so I dont needlessly step on chests and such, it fucks up when manually mining through the rocks and I'd have to jump in order to go up one block. 
Which while wearing the T3 mining suit makes me jump really high (This needs a toggle, btw)

So, basically if there is a trigger and it's something I mentioned, cool.
If Im wrong or theres more, I'd like to know the rest.
If it's a feature that you cannot step up in small spaces, It's dumb and you should change it, since it helps the most here.
And the super jump feature needs a toggle, maybe the same button as the jetpack.

**Comment by pWn3d1337** [comment]
I can't replicated your problem or don't undstand what you mean, The step assist works fine for me. can you post a screenshot where the step assist does not work, but you think it should?

As for toggle, yes this is an option to add one for the jumpheight bonus, but i don't want to spam keybinds/buttons too much. Adding it to the jetpack button is something I don't like.

**Comment by pWn3d1337** [comment]
oops, just wanted to comment, not comment and close.

**Comment by Fortist** [comment]
If there is a block that leads to a wall, it doesnt step up.
![2017-02-22_15 06 35](https://cloud.githubusercontent.com/assets/5784537/23230119/b33ea166-f910-11e6-8781-3130063ae6f9.png)

**Comment by Fortist** [comment]
I figured something out.
You cannot be looking at a block in order to step up. Ergo, if I were to be looking at the head height block, I wouldnt be able go up on the block below it

**Comment by pWn3d1337** [comment]
Everything works fine for me. Did you play in singleplayer or on a dedicated server? What version are you using?

**Comment by pWn3d1337** [comment]
No interaction for more than a month, closed.
